website/src/components/Chat/ChatConfigDesktop.tsx

```diff
@@ -14,6 +13,7 @@ export const ChatConfigDesktop = memo(function ChatConfigDesktop({ children }: {
       minH="0"
       flexDirection="column"
       minW="270px"
+      maxW="270px"
```

Comment:
if min and max are the same, can we just use `w`?

Reply:
Unfortunately it doesn't work lol